### PR TITLE
Mention devstats-query-data and devstats-data

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: content/analytics/index.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ title: "Analytics"
 type = 'card'
 header = '[devstats.scientific-python.org](https://devstats.scientific-python.org)'
 body = '''
-Generate developer statistics for a specified project using the [devstats](https://github.com/scientific-python/devstats) package.
+We generate developer statistics for various core packages using the [devstats](https://github.com/scientific-python/devstats) tool.
+[Data releases](https://github.com/scientific-python/devstats-data/releases) are made weekly.
```
Comment (bsipocz) [inline content/analytics/index.md:12]: Is this in place already?

File: content/github/index.md [hunk 2]
```diff
@@ -57,6 +57,13 @@ body = '''
 Add a GitHub status link directly to a CircleCI artifact.
 '''
 
+[[item]]
+type = 'card'
+header = '[devstats-query-action](https://github.com/scientific-python/devstats-query--action)'
```
Comment (bsipocz) [inline content/github/index.md:62]: typo
```suggestion
header = '[devstats-query-action](https://github.com/scientific-python/devstats-query-action)'
```

File: content/analytics/index.md (hunk 1 above)
Comment (stefanv) [reply content/analytics/index.md:12]: It is! Due to rate limiting, the action could not complete for all core projects. I'm looking into parallelization to see if that helps.

Comment (stefanv) [reply content/analytics/index.md:12]: https://github.com/scientific-python/devstats-data/actions/runs/11867300849

Comment (stefanv) [reply content/analytics/index.md:12]: First build is live! https://github.com/scientific-python/devstats-data/releases

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
Thanks @stefanv!

**Pull request merged**